packages/@azure/servicebus/data-plane/lib/queueClient.ts

```diff
@@ -208,16 +217,16 @@ export class QueueClient implements Client {
   // }
 
   /**
-   * Gets a SessionReceiver for receiving messages in batches or by registering handlers from a
+   * Creates a SessionReceiver for receiving messages in batches or by registering handlers from a
    * session enabled Queue. When no sessionId is given, a random session among the available
    * sessions is used.
-   *
+   * Throws error if an open receiver already exists for given sessionId.
    * @param options Options to provide sessionId and ReceiveMode for receiving messages from the
    * session enabled Servicebus Queue.
    *
    * @returns SessionReceiver An instance of a SessionReceiver to receive messages from the session.
    */
-  async getSessionReceiver(options?: SessionReceiverOptions): Promise<SessionReceiver> {
+  async createSessionReceiver(options?: SessionReceiverOptions): Promise<SessionReceiver> {
```

Comment:
`createReceiverForSession` ?

Reply:
I have a plan to consolidate `createReceiver` and this one, so this change (when I make it) will not be for long.
See https://github.com/Azure/azure-sdk-for-js/issues/1728